squire_lib/src/ffi_rounds.rs

```diff
@@ -0,0 +1,160 @@
+use crate::ffi::FFI_TOURNAMENT_REGISTRY;
+use crate::{
+    identifiers::{PlayerId, RoundId, TournamentId},
+    round::{Round, RoundStatus},
+};
+use std::alloc::{Allocator, Layout, System};
+use std::collections::HashSet;
+use std::option::Option;
+use std::ptr;
+use uuid::Uuid;
+
+impl RoundId {
+    /// Returns the round if it can be found in the tournament
+    fn get_tourn_round(self, tid: TournamentId) -> Option<Round> {
+        unsafe {
```

Comment:
This unsafe is unnecessary if you get right of the `get_mut`.

Reply:
Fixed in d33f556a8e954be45be0f6beecdbf19737bdbe46